R/getPrevalence.R

```diff
@@ -213,6 +212,14 @@ setMethod("getPrevalence", signature = c(x = "ANY"), function(
             stop("'sort' must be TRUE or FALSE.", call. = FALSE)
         }
         #
+        # Give warning if there are taxa with NA values
+        if( any( is.na(x) ) ){
+            msg <- paste0(
+                "The abundance table contains NA values and they are ",
+                ifelse(na.rm, "not", ""), "excluded (see 'na.rm').")
```

Comment:
is this in sync with the agg.na.rm?

Reply:
This is `na.rm` for `getPrevalence` function (`agg.na.rm` is for `agglomerateByRanks`) --> so yes

Reply:
Ok. Just ensure it is sufficiently clear from documentation what na.rm is for what purposes and there is no real risk for confusion between those.